lib/message.js

```diff
@@ -16,9 +16,9 @@ var $ = bitcore.util.preconditions;
  * constructs a new message to sign and verify.
  *
  * @param {String} message
- * @returns {Message}
```

Comment:
Hmm, that's right. In any case I thought that since this is a constructor it could be infered?